samtranslator/model/sam_resources.py

```diff
@@ -2193,18 +2203,20 @@ def _create_logging_default(self) -> Tuple[LogConfigType, IAMRole]:
 
         return log_config, cloudwatch_role
 
-    def _parse_logging_properties(self) -> Tuple[LogConfigType, Optional[IAMRole]]:
+    def _parse_logging_properties(self, model: GraphQLApiProperties) -> Tuple[LogConfigType, Optional[IAMRole]]:
         """Parse logging properties from SAM template, and use defaults if required keys dont exist."""
-        if not isinstance(self.Logging, dict):
+        if not isinstance(model.Logging, GraphQLApiLogging):
             return self._create_logging_default()
 
         log_config: LogConfigType = {}
 
-        if "ExcludeVerboseContent" in self.Logging:
-            log_config["ExcludeVerboseContent"] = self.Logging["ExcludeVerboseContent"]
+        if model.Logging.ExcludeVerboseContent:
+            log_config["ExcludeVerboseContent"] = cast(
+                bool, model.Logging.ExcludeVerboseContent
+            )  # TODO: better handling PassThroughProp
 
-        log_config["FieldLogLevel"] = self.Logging.get("FieldLogLevel", "ALL")
-        log_config["CloudWatchLogsRoleArn"] = self.Logging.get("CloudWatchLogsRoleArn", None)
+        log_config["FieldLogLevel"] = model.Logging.FieldLogLevel or "ALL"
+        log_config["CloudWatchLogsRoleArn"] = model.Logging.CloudWatchLogsRoleArn
```

Comment:
So if `CloudWatchLogsRoleArn` isn't defined in template, then in `model.Logging.CloudWatchLogsRoleArn` is set to `None`?

Reply:
`CloudWatchLogsRoleArn=None ExcludeVerboseContent=PassThroughProp(__root__=True) FieldLogLevel='ERROR'`